packages/server/src/auth/auth.service.ts

```diff
@@ -130,8 +133,10 @@ export class AuthService {
       const user = await this.userService.createUser(data);
       const payload = { id: user.id, projectId: user.projectId, role: user.role };
       const resp = { accessToken: this.jwtService.sign(payload, { expiresIn: process.env.JWT_EXPIRATION }) };
+      this.logger.log(`User of user id: ${user.id} Created`);
       return resp;
     } catch (err) {
+      this.logger.log(err);
```

Comment:
If error we want to use error instead of log:
```suggestion
      this.logger.error(err);
```